### PR TITLE
fix: v8 Calendar focus outline clipping in HCM

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

Since cells are positioned and give an z-index value, the focus indicator outline in high contrast mode was getting clipped by cells after it in the DOM:
image

## New Behavior

This gives cells a transparent bg in high contrast mode, so the focus and hover indicators do not get clipped. The hover and pressed styles keep their bg, so when a cell is hovered, it will still (intentionally) override any adjacent cell outlines (e.g. so hovering a cell next to a focused cell doesn't give you weird overlapping double lines):
image
image

## Related Issue(s)

* Fixes [15811](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/15811)

### CODE REVIEW

File: change/@fluentui-react-731171c2-4a2b-4484-956b-667804fe01e2.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-731171c2-4a2b-4484-956b-667804fe01e2.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26723&commitId=2915193da8d8eaa49c92d920d20cd6e7cf2d9f50&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0b35790d67cad33bd1d52d2418fcca6ef8b4a710'>0b35790d67cad33bd1d52d2418fcca6ef8b4a710</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount_base.html">2256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount.html">2541</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount_base.html">98845</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount.html">101103</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/BaseButton-mount_base.html">1431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/BaseButton-mount.html">1463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Breadcrumb-mount_base.html">3535</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Breadcrumb-mount.html">3614</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Checkbox-mount_base.html">3190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Checkbox-mount.html">3127</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/CheckboxBase-mount_base.html">2908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/CheckboxBase-mount.html">2828</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ChoiceGroup-mount_base.html">5384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ChoiceGroup-mount.html">5374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ComboBox-mount_base.html">1470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ComboBox-mount.html">1548</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/CommandBar-mount_base.html">11522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/CommandBar-mount.html">11419</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ContextualMenu-mount_base.html">16304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ContextualMenu-mount.html">16340</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DefaultButton-mount_base.html">1670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DefaultButton-mount.html">1683</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRow-mount_base.html">4129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRow-mount.html">4168</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRowFast-mount_base.html">4115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRowFast-mount.html">4159</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRowNoStyles-mount_base.html">4127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DetailsRowNoStyles-mount.html">4059</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Dialog-mount_base.html">3645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Dialog-mount.html">3721</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DocumentCardTitle-mount_base.html">705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/DocumentCardTitle-mount.html">687</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Dropdown-mount_base.html">3862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Dropdown-mount.html">3865</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/FocusTrapZone-mount_base.html">2414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/FocusTrapZone-mount.html">2394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/FocusZone-mount_base.html">2336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/FocusZone-mount.html">2321</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount_base.html">2256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount.html">2541</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-virtual-rerender_base.html">1363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-virtual-rerender.html">1341</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedList-virtual-rerender-with-unmount.html">1941</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-mount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-mount.html">680</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-virtual-rerender_base.html">665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-virtual-rerender.html">642</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">686</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">697</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/IconButton-mount_base.html">2283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/IconButton-mount.html">2312</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Label-mount_base.html">882</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Label-mount.html">881</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Layer-mount_base.html">5185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Layer-mount.html">5286</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Link-mount_base.html">997</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Link-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/MenuButton-mount_base.html">2036</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/MenuButton-mount.html">2056</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/MessageBar-mount_base.html">2840</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/MessageBar-mount.html">2714</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Nav-mount_base.html">3892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Nav-mount.html">3880</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/OverflowSet-mount_base.html">1610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/OverflowSet-mount.html">1639</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Panel-mount_base.html">3041</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Panel-mount.html">3095</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Persona-mount_base.html">1551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Persona-mount.html">1566</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Pivot-mount_base.html">1960</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Pivot-mount.html">1935</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/PrimaryButton-mount_base.html">1817</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/PrimaryButton-mount.html">1827</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Rating-mount_base.html">8197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Rating-mount.html">8427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SearchBox-mount_base.html">1821</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SearchBox-mount.html">1874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Shimmer-mount_base.html">3364</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Shimmer-mount.html">3463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Slider-mount_base.html">2493</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Slider-mount.html">2463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SpinButton-mount_base.html">5475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SpinButton-mount.html">5657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Spinner-mount_base.html">957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Spinner-mount.html">959</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SplitButton-mount_base.html">3652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SplitButton-mount.html">3716</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Stack-mount_base.html">976</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Stack-mount.html">976</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/StackWithIntrinsicChildren-mount_base.html">2654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/StackWithIntrinsicChildren-mount.html">2754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/StackWithTextChildren-mount_base.html">5691</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/StackWithTextChildren-mount.html">5701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SwatchColorPicker-mount_base.html">12286</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/SwatchColorPicker-mount.html">12428</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TagPicker-mount_base.html">3141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TagPicker-mount.html">3280</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount_base.html">98845</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount.html">101103</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TeachingBubble-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Text-mount_base.html">915</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Text-mount.html">955</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TextField-mount_base.html">1884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/TextField-mount.html">1859</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-mount_base.html">1897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-mount.html">1837</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-virtual-rerender_base.html">1267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-virtual-rerender.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2548</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Toggle-mount_base.html">1343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/Toggle-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/buttonNative-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26723/perf-test/buttonNative-mount.html">655</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by khmakoto — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**